src/main/kotlin/com/ioki/lokalise/gradle/plugin/tasks/UploadTranslationsTask.kt

```diff
@@ -1,10 +1,7 @@
 package com.ioki.lokalise.gradle.plugin.tasks
 
 import com.ioki.lokalise.api.models.FileUpload
-import com.ioki.lokalise.gradle.plugin.LokaliseExtension
-import com.ioki.lokalise.gradle.plugin.FileInfo
-import com.ioki.lokalise.gradle.plugin.LokaliseApi
-import com.ioki.lokalise.gradle.plugin.LokaliseApiFactory
+import com.ioki.lokalise.gradle.plugin.*
```

Comment:
Please don't use a star import 🙈

Reply:
I did it here: https://github.com/ioki-mobility/LokaliseGradlePlugin/pull/73/commits/f3489c04dab8f09b1e21b913233960b4ef6bf6f4